osu.Game/Overlays/Comments/CommentEditor.cs

```diff
@@ -1,48 +1,52 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-#nullable disable
-
 using osu.Framework.Allocation;
-using osu.Framework.Graphics.Containers;
+using osu.Framework.Bindables;
 using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Localisation;
 using osu.Game.Graphics;
 using osu.Game.Graphics.Sprites;
-using osu.Framework.Graphics.UserInterface;
-using osu.Framework.Graphics.Sprites;
-using osuTK.Graphics;
 using osu.Game.Graphics.UserInterface;
-using System.Collections.Generic;
-using System;
+using osu.Game.Graphics.UserInterfaceV2;
 using osuTK;
-using osu.Framework.Bindables;
+using osuTK.Graphics;
 
 namespace osu.Game.Overlays.Comments
 {
     public abstract partial class CommentEditor : CompositeDrawable
     {
         private const int side_padding = 8;
 
-        public Action<string> OnCommit;
+        protected abstract LocalisableString FooterText { get; }
 
-        public bool IsLoading
-        {
-            get => commitButton.IsLoading;
-            set => commitButton.IsLoading = value;
-        }
+        protected abstract LocalisableString CommitButtonText { get; }
 
-        protected abstract string FooterText { get; }
+        protected abstract LocalisableString TextBoxPlaceholder { get; }
 
-        protected abstract string CommitButtonText { get; }
+        protected FillFlowContainer ButtonsContainer { get; private set; } = null!;
 
-        protected abstract string TextBoxPlaceholder { get; }
+        protected readonly Bindable<string> Current = new Bindable<string>();
```

Comment:
```suggestion
        protected readonly Bindable<string> Current = new Bindable<string>(string.Empty);
```

Otherwise the default is `null`. Unfortunate shortcoming of bindables but it is what it is.